src/components/start/sidepane.scss

```diff
@@ -195,6 +195,11 @@ body[data-theme="dark"] .sidePane {
   z-index: 9999;
   border-radius: 8px;
 
```

Comment:
Add `max-height:500px` here

Reply:
this pr has nothing to do with the sidepane

Reply:
oh wait

Reply:
yes

Reply:
apply this for `.calnpane`

Reply:
thank you it works now